.circleci/config.templ.yml

```diff
@@ -16,7 +16,7 @@ mongo_image: &mongo_image mongo:3.6@sha256:19c11a8f1064fd2bb713ef1270f79a742a184
 httpbin_image: &httpbin_image kennethreitz/httpbin@sha256:2c7abc4803080c22928265744410173b6fea3b898872c01c5fd0f0f9df4a59fb
 vertica_image: &vertica_image sumitchawla/vertica:latest
 rabbitmq_image: &rabbitmq_image rabbitmq:3.7-alpine
-testagent_image: &testagent_image ghcr.io/datadog/dd-apm-test-agent/ddapm-test-agent:v1.11.0
+testagent_image: &testagent_image williamconti549/dd-apm-test-agent:integration-versions
```

Comment:
Is this just a placeholder? We should get a new release of the test agent before merging this PR. Can we call out this dependency in the PR description?

Reply:
this was just testing for a future project and is not necessary for this PR, will revert